src/mysql.php

```diff
@@ -1,15 +1,16 @@
 <?php
 /**
- * php7-mysql-shim
+ * php7-mysql-shim.
```

Comment:
This is a title, shouldn't end in a period IMO

Reply:
This is job of fixer.